test/valgrind/valgrindTestSuite.sh

```diff
@@ -590,7 +634,10 @@ then
 
     if [ "$lost" != "0" ]
     then
-      failedTest "test/valgrind/$vtest.*" $vtest $lost "valgrind" $xTestNo
+      leakFound "test/valgrind/$vtest.*" $vtest $lost "valgrind" $xTestNo
```

Comment:
Yes, probably it is better to remove the whole .vtest thing. However, not in this PR.

In the meanwhile (i.e. in the context of this PR) a warning comment could be useful, e.g. "The .vtest test are deprectaed, so memory corruption is not done for them, only memory errors. Probably this is going to be removed compeltely in a next refactor".